diff --git a/node_modules/react-swipeable-views/lib/SwipeableViews.js b/node_modules/react-swipeable-views/lib/SwipeableViews.js
index cdcfaf9..387d9eb 100644
--- a/node_modules/react-swipeable-views/lib/SwipeableViews.js
+++ b/node_modules/react-swipeable-views/lib/SwipeableViews.js
@@ -623,22 +623,21 @@ function (_React$Component) {
           updateHeight: this.updateHeight
         });
       }
-    } // eslint-disable-next-line camelcase,react/sort-comp
-
+    }
   }, {
-    key: "UNSAFE_componentWillReceiveProps",
-    value: function UNSAFE_componentWillReceiveProps(nextProps) {
-      var index = nextProps.index;
+    key: "componentDidUpdate",
+    value: function componentDidUpdate(prevProps) {
+      var index = this.props.index;
 
-      if (typeof index === 'number' && index !== this.props.index) {
+      if (typeof index === 'number' && index !== prevProps.index) {
         if (process.env.NODE_ENV !== 'production') {
-          (0, _reactSwipeableViewsCore.checkIndexBounds)(nextProps);
+          (0, _reactSwipeableViewsCore.checkIndexBounds)(this.props);
         }
 
         this.setIndexCurrent(index);
         this.setState({
           // If true, we are going to change the children. We shoudn't animate it.
-          displaySameSlide: (0, _reactSwipeableViewsCore.getDisplaySameSlide)(this.props, nextProps),
+          displaySameSlide: (0, _reactSwipeableViewsCore.getDisplaySameSlide)(prevProps, this.props),
           indexLatest: index
         });
       }
diff --git a/node_modules/react-swipeable-views/src/SwipeableViews.js b/node_modules/react-swipeable-views/src/SwipeableViews.js
index 219e215..a2e362a 100644
--- a/node_modules/react-swipeable-views/src/SwipeableViews.js
+++ b/node_modules/react-swipeable-views/src/SwipeableViews.js
@@ -299,19 +299,18 @@ class SwipeableViews extends React.Component {
     }
   }
 
-  // eslint-disable-next-line camelcase,react/sort-comp
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    const { index } = nextProps;
+  componentDidUpdate(prevProps) {
+    const { index } = this.props;
 
-    if (typeof index === 'number' && index !== this.props.index) {
+    if (typeof index === 'number' && index !== prevProps.index) {
       if (process.env.NODE_ENV !== 'production') {
-        checkIndexBounds(nextProps);
+        checkIndexBounds(this.props);
       }
 
       this.setIndexCurrent(index);
       this.setState({
         // If true, we are going to change the children. We shoudn't animate it.
-        displaySameSlide: getDisplaySameSlide(this.props, nextProps),
+        displaySameSlide: getDisplaySameSlide(prevProps, this.props),
         indexLatest: index,
       });
     }
