diff --git a/node_modules/react-swipeable-views-utils/lib/autoPlay.js b/node_modules/react-swipeable-views-utils/lib/autoPlay.js
index 79e78b0..77e41ae 100644
--- a/node_modules/react-swipeable-views-utils/lib/autoPlay.js
+++ b/node_modules/react-swipeable-views-utils/lib/autoPlay.js
@@ -121,19 +121,16 @@ function autoPlay(MyComponent) {
       } // eslint-disable-next-line camelcase,react/sort-comp
 
     }, {
-      key: "UNSAFE_componentWillReceiveProps",
-      value: function UNSAFE_componentWillReceiveProps(nextProps) {
-        var index = nextProps.index;
+      key: "componentDidUpdate",
+      value: function componentDidUpdate(prevProps) {
+        var index = this.props.index;
 
-        if (typeof index === 'number' && index !== this.props.index) {
+        if (typeof index === 'number' && index !== prevProps.index) {
           this.setState({
             index: index
           });
         }
-      }
-    }, {
-      key: "componentDidUpdate",
-      value: function componentDidUpdate(prevProps) {
+
         var shouldResetInterval = !(0, _shallowEqual.shallowEqualObjects)({
           index: prevProps.index,
           interval: prevProps.interval,
diff --git a/node_modules/react-swipeable-views-utils/src/autoPlay.js b/node_modules/react-swipeable-views-utils/src/autoPlay.js
index 48d7976..ce58ce9 100644
--- a/node_modules/react-swipeable-views-utils/src/autoPlay.js
+++ b/node_modules/react-swipeable-views-utils/src/autoPlay.js
@@ -20,18 +20,15 @@ export default function autoPlay(MyComponent) {
       this.startInterval();
     }
 
-    // eslint-disable-next-line camelcase,react/sort-comp
-    UNSAFE_componentWillReceiveProps(nextProps) {
-      const { index } = nextProps;
+    componentDidUpdate(prevProps) {
+      const { index } = this.props;
 
-      if (typeof index === 'number' && index !== this.props.index) {
+      if (typeof index === 'number' && index !== prevProps.index) {
         this.setState({
           index,
         });
       }
-    }
 
-    componentDidUpdate(prevProps) {
       const shouldResetInterval = !shallowEqualObjects(
         {
           index: prevProps.index,
